roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -110,6 +110,36 @@
 - name: Configure swap
   ansible.builtin.import_tasks: swap.yml
 
+- name: Check if CA cert file exists
+  delegate_to: localhost
+  ansible.builtin.stat:
+    path: /var/lib/openstack/cacerts/bootstrap/tls-ca-bundle.pem
+  register: ca_file
+
+- name: Copy CA cert file if present
+  when: ca_file.stat.exists
+  become: true
+  block:
+    - name: Ensure that the CA destination directory exists
+      ansible.builtin.file:
+        path: "/etc/pki/ca-trust/source/anchors"
+        state: "directory"
+        setype: "cert_t"
+        owner: "root"
+        group: "root"
+        mode: "0755"
+
+    - name: Copy CA certs to the standard location on the compute node
+      ansible.builtin.copy:
+        src: "/var/lib/openstack/cacerts/bootstrap/tls-ca-bundle.pem"
+        dest: "/etc/pki/ca-trust/source/anchors/tls-ca-bundle.pem"
+        mode: '0644'
+        owner: root
+        group: root
+
+    - name: Run update-ca-trust on the compute node
+      ansible.builtin.command: /usr/bin/update-ca-trust
```

Comment:
can we check if the previous task actually changed the file? b/c otherwise, we don't need to run update-ca-trust every time.

Reply:
ack - latest version adds a notify: which will invoke a handler to do the update-ca-trust on on changes.